electron/js/menu/system.js

```diff
@@ -267,6 +267,18 @@ var win32Template = {
   ],
 };
 
+var linuxTemplate = {
+  label: config.NAME,
```

Comment:
I guess so, though it isn't really a 'traditional' menu, is it? 
To change that, would line 271 change to `i18n: 'menuFile',`?